spring-kafka/src/main/java/org/springframework/kafka/retrytopic/FixedDelayStrategy.java

```diff
@@ -22,8 +22,10 @@
  *
  * @author Tomaz Fernandes
  * @since 2.7
+ * @deprecated in a future release, will be replaced by {@link SameIntervalTopicReuseStrategy}.
  *
  */
+@Deprecated(forRemoval = true) // in 3.1
```

Comment:
Hi João! I'm not sure I understand - as far as I am aware we won't remove anything in 3.1, as we need to keep backwards compatibility until the next major or at least a few minors. Care to elaborate on the comment? Thanks!

Reply:
I followed this:
https://github.com/spring-projects/spring-kafka/pull/2497#discussion_r1093411586

Anyway, I can remove the removal mentions, just want to hear from @garyrussell again.

Reply:
Oh, I didn't remember this comment. Perhaps @garyrussell can enlighten us on this?

Reply:
My current plan is to look at this tomorrow, unless something more urgent comes in.

I am not opposed to removing stuff in the next minor release, it really depends on how critical or invasive it is.